packages/react-client/src/ReactFlightClient.js

```diff
@@ -34,7 +34,7 @@ export type JSONValue =
   | null
   | boolean
   | string
-  | {+[key: string]: JSONValue}
+  | {+[key: string]: JSONValue, ...}
```

Comment:
This should be exact. Otherwise it has no meaning. Since it should restrict to only properties with JSONValue.

Reply:
fixes